doc/active_docs/Account Management API.json

```diff
@@ -7506,6 +7506,36 @@
         }
       ]
     },
+    {
+      "path": "/admin/api/services/{service_id}/proxy/deploy.xml",
+      "responseClass": "proxy",
+      "operations": [
+        {
+          "httpMethod": "POST",
+          "summary": "Proxy Deploy",
+          "description": "Creates a new APIcast configuration version for the Staging environment (production environment in case of Service Mesh)",
```

Comment:
not sure i understand why we have 2 different descriptions: here and proxies contr L:84

Reply:
Commit batch suggestion did not work ... gosh